apps/web/src/modules/profile/settings/settings.page.tsx

```diff
@@ -18,10 +16,10 @@ export const SettingsPage: Component = () => {
     document.documentElement.classList.add(isDarkMode() ? 'dark' : 'light');
   };
 
-  const hasRequestsFlag = hasFeatureFlag(Feature.requests);
+  const hasRequestsFlag = hasFeatureFlag(Feature.events);
```

Comment:
Please, rename the variable accordingly to new flagged feature: events

Reply:
Good catch :100: